features/proxy_config.feature

```diff
@@ -42,23 +43,17 @@ Feature: Proxy configuration
         esm-infra     +yes      +enabled      +UA Infra: Extended Security Maintenance \(ESM\)
```

Comment:
This slipped in in a previous PR but would you mind fixing it here?
This line only passes on xenial.
So we need a change like this:
```diff
         And I run `route del default` with sudo
         And I attach `contract_token` with sudo
+        # Just to verify that the machine is attached
+        Then stdout matches regexp:
+        """
+        This machine is now attached
+        """
-        And I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
+        When I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
         Then stdout matches regexp:
         """
        .*CONNECT contracts.canonical.com.*
         """
-        When I run `ua status` with sudo
-        # Just to verify that the machine is attached
-        Then stdout matches regexp:
-        """
-        esm-infra     +yes      +enabled      +UA Infra: Extended Security Maintenance \(ESM\)
-        """
```

Reply:
And we'll need the same change down in the new "manual with config" scenario below

Reply:
Actually, this may be indicative of an actual bug unrelated to this PR. So don't make this change, we'll need to investigate this.

Reply:
Okay, feels like I am listening to your thoughts 😂 😂.